generator/templates/structfield.gotmpl

```diff
@@ -12,6 +12,7 @@
     {{- template "structTag" dict "Tag" "json" "Schema" $prop }}
   {{- end }}
   {{- if .XMLName }} xml:"{{ .XMLName }}"{{ end }}
+  {{- if .Example }} example:"{{ .Example }}"{{ end }}
```

Comment:
I think this should probably be escaped, an example could contain a double quote in theory. But I wasn't sure which filter to use for this.